website/source/docs/job-specification/sidecar_task.html.md

```diff
@@ -18,9 +18,10 @@ description: |-
   </tr>
 </table>
 
-The `sidecar_task` stanza allows configuring various options for
-the sidecar proxy managed by Nomad for Consul Connect integration.
-It is valid only within the context of a connect stanza.
+The `sidecar_task` stanza allows configuring various options for the proxy
```

Comment:
should we talk about what options? If this is resources and other task specific configuration, how about something like: 
"The `sidecar_task` stanza allows configuring runtime options for the proxy sidecar task, such as resource requirements, kill timeouts and more as defined below"